### DISCUSSION TITLE
Multiple node activities

### DISCUSSION THREAD

**Discussion by pkwah** [microsoft/vscode-cpptools#12419] [Q&A] [answered]
Hello

I am relatively new to VS. When I open VS code and connect to a server, I notice that I have multiple node activities and jobs labeled R even without running anything. How can I stop this? I am using a shared server, and these activities keep showing up and probably using memory. I have shared a screenshot below. Apologies. I tried searching for ideas, but I could not figure out how to fix it.

![image](https://github.com/microsoft/vscode-cpptools/assets/51105310/f5e0668c-7930-46cb-9a0f-53cf36ce516f)

**Comment by bobbrow** [comment, marked as answer]
Hi @pkwah,

Our extension spawns processes called `cpptools` and `cpptools-srv`. I don't know who owns those `node` or `R` processes.  It could be VS Code or another extension.  Could you ask here -> https://stackoverflow.com/questions/tagged/visual-studio-code?

**Reply by pkwah** [reply]
Thanks @bobbrow
